typescript/cli/src/config/warp.ts

```diff
@@ -44,17 +43,17 @@ export const WarpRouteConfigSchema = z.object({
     .nonempty(),
 });
 
-type InferredType = z.infer<typeof WarpRouteConfigSchema>;
+type InferredType = z.infer<typeof WarpDeployConfigSchema>;
 // A workaround for Zod's terrible typing for nonEmpty arrays
-export type WarpRouteConfig = {
+export type WarpDeployConfig = {
   base: InferredType['base'];
   synthetics: Array<InferredType['synthetics'][0]>;
 };
 
-export function readWarpRouteConfig(filePath: string) {
+export function readWarpDeployConfig(filePath: string) {
   const config = readYamlOrJson(filePath);
-  if (!config) throw new Error(`No warp config found at ${filePath}`);
-  const result = WarpRouteConfigSchema.safeParse(config);
+  if (!config) throw new Error(`No warp deploy config found at ${filePath}`);
```

Comment:
in some places we use "warp deploy config" and others "warp route deployment config", think it makes sense to unify everything to the latter form for clarity

Reply:
probably want to do that with variable/function naming too 
e.g. 
- warpDeploymentConfigPath -> warpRouteDeploymentConfigPath
- createWarpDeployConfigCommand -> createWarpRouteDeploymentConfigCommand

thoughts?

Reply:
Agreed, I'll do that now